monkey/agent_plugins/util.sh

```diff
@@ -33,12 +33,11 @@ lower() {
 get_plugin_filename() {
     _plugin_path=${1:-"."}
 
-    manifest_filename=$(get_plugin_manifest_filename "$_plugin_path")
+    _manifest_filename=$(get_plugin_manifest_filename "$_plugin_path")
+
+    _name=$(get_plugin_name "${_plugin_path}" "${_manifest_filename}")
+    _type=$(get_plugin_type "${_plugin_path}" "${_manifest_filename}")
```

Comment:
NIT: We could just pass the path to the manifest file, rather than passing both the plugin path and the manifest filename